aries-site/src/data/structures/components.js

```diff
@@ -100,6 +100,26 @@ export const components = [
       background: 'background-front',
     },
   },
+  {
+    name: 'FileInput',
+    category: 'Inputs',
+    description:
+      'A widget which allows the user to select one or more files to upload.',
```

Comment:
```suggestion
      'A file upload control that can be used to upload one or more files.',
```
Aligning with Figma thumbnail.